db/migrate/20180911195906_add_genre_id_to_performer.rb

```diff
@@ -0,0 +1,5 @@
+class AddGenreIdToPerformer < ActiveRecord::Migration[5.2]
+  def change
+    add_column :performers, :genre_id, :string
```

Comment:
Right, it IS an association. In the seed file you are setting it as a string value. Please review that.